main/modes/jumper/mode_jumper.c

```diff
@@ -1810,20 +1810,20 @@ void jumperDrawHud(display_t* d, font_t* prompt, font_t* font, font_t* outline)
 
     if (j->scene->level < 10)
     {
-        drawText(d, font, c555, textBuffer, 190, 220);
+        drawText(d, font, c555, textBuffer, 180, 220);
     }
     else
     {
-        drawText(d, font, c555, textBuffer, 174, 220);
+        drawText(d, font, c555, textBuffer, 164, 220);
     }
 
     drawText(d, prompt, c555, "SCORE", 28, 10);
     snprintf(textBuffer, sizeof(textBuffer) - 1, "%u", j->scene->score);
     drawText(d, prompt, c555, textBuffer, 28, 30);
 
-    drawText(d, prompt, c555, "HI SCORE", 168, 10);
+    drawText(d, prompt, c555, "HI SCORE", d->w - textWidth(prompt, "HI SCORE") - 18, 10);
     snprintf(textBuffer, sizeof(textBuffer) - 1, "%u", j->highScore);
-    drawText(d, prompt, c555, textBuffer, 204, 30);
+    drawText(d, prompt, c555, textBuffer, 202, 30);
```

Comment:
Why is the score not getting measured and right justified? Or if it's supposed to be left justified, why not use the same value for X as "HI SCORE"?

Reply:
Good question!

If we come up with a maximum score, it would make sense to right-justify both scores, to keep the digits in the same places as you gain points.

If we can't come up with a reasonable max score, the left score should be left-justified, and the right high score should be right-justified, to give them the most room to grow.

Maximum that the ESP can store in NVS appears to be 2,147,483,647, I'll give it a whirl to see if that fits on the screen twice comfortably

Reply:
Right-justifying the score to keep the digits in one place only works if the font is monospaced 😭 

Switched to left-justify, and added some code to change to a smaller font if anyone gets too wide of a score. Tested both cases and the transition on emu, and tested the 99.999%-case on hardware